src/backend/src/services/change-request.services.ts

```diff
@@ -0,0 +1,556 @@
+   * @param workPackageNumber  the work package number for the wbs element
+   * @param type  the type of cr
+   * @param what  the description of the change
+   * @param why  the reason for the change
+   * @param budgetImpact  the impact on the budget
+   * @returns  the id of the created cr
+   * @throws if user is not allowed to create crs, if wbs element does not exist, or if the cr type is not standard
+   */
+  static async createStandardChangeRequest(
+    submitter: User,
+    carNumber: number,
+    projectNumber: number,
+    workPackageNumber: number,
+    type: CR_Type,
+    what: string,
+    why: any,
+    budgetImpact: number
+  ): Promise<Number> {
+    // verify user is allowed to create stage gate change requests
+    if (submitter.role === Role.GUEST) throw new AccessDeniedException();
+
+    // verify wbs element exists
+    const wbsElement = await prisma.wBS_Element.findUnique({
+      where: {
+        wbsNumber: {
+          carNumber,
+          projectNumber,
+          workPackageNumber
+        }
+      }
+    });
+
+    if (!wbsElement) throw new NotFoundException('WBS Element', `${carNumber}.${projectNumber}.${workPackageNumber}`);
+
+    const createdCR = await prisma.change_Request.create({
+      data: {
+        submitter: { connect: { userId: submitter.userId } },
+        wbsElement: { connect: { wbsElementId: wbsElement.wbsElementId } },
+        type,
+        scopeChangeRequest: {
+          create: {
+            what,
+            scopeImpact: '',
+            timelineImpact: 0,
+            budgetImpact: 0,
+            why: { createMany: { data: why } }
+          }
+        }
+      },
+      include: {
+        wbsElement: {
+          include: {
+            project: { include: { team: { include: { leader: true } }, wbsElement: true } },
+            workPackage: {
+              include: {
+                project: { include: { team: { include: { leader: true } }, wbsElement: true } }
+              }
+            }
+          }
+        }
+      }
+    });
+
+    const project = createdCR.wbsElement.workPackage?.project || createdCR.wbsElement.project;
+    if (project?.team) {
+      const slackMsg = `${type} CR submitted by ${submitter.firstName} ${submitter.lastName} for the ${project.wbsElement.name} project`;
+      await sendSlackChangeRequestNotification(project.team, slackMsg, createdCR.crId, budgetImpact);
+    }
+    return createdCR.crId;
+  }
+
+/**
+ * 
+ * @param submitter  The user creating the cr
+ * @param crId  the id of the change request
+ * @param budgetImpact  the impact on the budget
+ * @param description  the description of the proposed solution
+ * @param timelineImpact  the impact on the timeline
+ * @param scopeImpact  the impact on the scope
+ * @returns  the id of the created cr
+ * @throws if user is not allowed to create crs, if the change request is not found, or if the change request has already been reviewed
+ */
```

Comment:
these comments are misaligned, run prettier on this file. why does your prettier keep getting turned off?